### PR TITLE
Add null check class

### PR DESCRIPTION
**Pull request by willsb**
Squash whenever

### CODE REVIEW

File: Toggl.Ultrawave/Helper/Ensure.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Toggl.Ultrawave.Helper
```
Comment (amulware) [inline Toggl.Ultrawave/Helper/Ensure.cs]: Should something like this be in the wave itself? Will it only be used there, and in none of the other projects?

Comment (willsb) [inline Toggl.Ultrawave/Helper/Ensure.cs]: Excellent point. I think that, dependency wise, Ultrawave is the outermost project and won't depend on anything. Unless we have a common "extensions" project that's reused by anyone. Not sure how I feel about that, but it's sure very helpful

**Comment by amulware** [comment]
We decided to put helpers and extensions like this into a separate project.

**Comment by willsb** [comment]
We can't forget to delete the branch when we close PRs without merging

**Pull request closed without merging**